third-party/linshare/src/main/java/org/apache/james/linshare/client/GrantUserPermissionRequest.java

```diff
@@ -1,6 +1,5 @@
 package org.apache.james.linshare.client;
```

Comment:
It looks like this commit is somehow a fixup of the previous one. We don't usually keep all the steps of our history, we only keep steps that make the reading easier and we merge commits when that are parts of a single step.